hack/os-packages/rpm-fips-artifacts-getter/centos/7/containerd.fips.repo

```diff
@@ -0,0 +1,5 @@
+[containerd-fips]
+baseurl = https://containerd-fips.s3.us-east-2.amazonaws.com/7/x86_64
+gpgcheck = 0
+gpgkey = ""
```

Comment:
Help needed: Unable to find out gpgkey URL for this repo. 
Should we change the baseURL to point to `packages.d2iq.com` for the FIPS packages for both kuberentes and containerd rpms ?

Reply:
thanks for providing link to gpgkey